engine/lite/k8sDeploymentManager/src/main/scala/pl/touk/nussknacker/k8s/manager/deployment/DeploymentUtils.scala

```diff
@@ -0,0 +1,21 @@
+package pl.touk.nussknacker.k8s.manager.deployment
+
+import com.typesafe.config.{Config, ConfigFactory, ConfigRenderOptions}
+import org.apache.commons.io.IOUtils
+import play.api.libs.json.Json
+import skuber.apps.v1.Deployment
+
+object DeploymentUtils {
+
+  def createDeployment(deploymentString: String): Deployment = {
+    val deploymentSpec = Json.parse(deploymentString).as[Deployment]
+    deploymentSpec
+  }
+
+  def createDeployment(config: Config): Deployment = {
```

Comment:
1. can methods be ordered how it's are invoking themselves? I recommend this pattern - it is easier to read from top to down instead of "jumping" between methods
2. Why you read this stream to string instead of passing url or stream?
3. Why json instead of conf? You don't want to use there concise hocon format?
4. if you apply comment above about `minReadySeconds` and 2. this method be changed to something like `parseDeploymentWithFallback` taking resource (stream or url) as a parameter, and `deploymentMinimal.json` will be renamed to `defaultMinimalDeployment.json`. Do you agree?

Reply:
1. sorted 2. used parseUrl, 3. changed to conf, 4 changed